lstchain/calib/camera/__init__.py

```diff
@@ -5,8 +5,10 @@
 """
 
 from .calib import lst_calibration, load_calibrator_from_config
+from .r0 import CameraR0Calibrator
```

Comment:
why do you need to include it here? was it called by any function

Reply:
Yes, by one of the scripts. I discovered it through the tests with `--help`

Reply:
I can remove it here and change the relevant import if you prefer that.

Reply:
no, I'm fine, I was just wondering why it did not fire up our tests, but makes sense.